re.c

```diff
@@ -3764,19 +3786,20 @@ rb_check_regexp_type(VALUE re)
 
 /*
  *  call-seq:
- *     Regexp.try_convert(obj) -> re or nil
+ *    Regexp.try_convert(object) -> regexp or nil
+ *
+ *  Returns +object+ if it is a regexp:
+ *
+ *    Regexp.try_convert(/re/) # => /re/
  *
- *  Try to convert <i>obj</i> into a Regexp, using to_regexp method.
- *  Returns converted regexp or nil if <i>obj</i> cannot be converted
- *  for any reason.
+ *  Otherwise if +object+ responds to <tt>:to_hash</tt>,
```

Comment:
All of the `to_hash` in this method documentation needs to be switched to `to_regexp`.

Reply:
Fixed.